cg/meta/report/rnafusion.py

```diff
@@ -47,3 +69,45 @@ def get_sample_metadata(
             rin=self.lims_api.get_sample_rin(sample_id=sample.internal_id),
             uniquely_mapped_reads=sample_metrics.uniquely_mapped_percent,
         )
+
+    def get_genome_build(self, analysis_metadata: AnalysisModel) -> str:
+        """Returns the build version of the genome reference of a specific case."""
+        return GenomeVersion.hg38.value
```

Comment:
This one looks a bit strange to me. It does not seem to use the parameters supplied and returns a constant. Do we need it or can we just use the constant directly?

Reply:
Considering that `get_genome_build` is also invoked from a method in the general `report_api`, I believe it would be cleaner to keep it structured in this way